build.js

```diff
@@ -60,14 +68,47 @@ StyleDictionaryPackage.registerFilter({
   name: 'excludeBrand',
   matcher: function (token) {
     return token.category !== 'brand';
-  }
+  },
 });
 
+/**
+ * REGISTER FORMATS
+ * @see https://amzn.github.io/style-dictionary/#/api?id=registerformat
+ */
+
 StyleDictionaryPackage.registerFormat({
   name: 'json/flat',
   formatter: function (dictionary) {
     return JSON.stringify(dictionary.allProperties, null, 2);
-  }
+  },
+});
+
+StyleDictionaryPackage.registerFormat({
+  name: 'javascript/nested',
+  formatter: function (dictionary) {
+    const tokens = dictionary.properties;
+    const removeMetadataAndFlatten = (jsonObj) => {
+      for (var key in jsonObj) {
+        if (typeof jsonObj[key] == 'object') {
+          if (jsonObj[key].hasOwnProperty('value')) {
+            jsonObj[key] = jsonObj[key]['value'];
+          } else {
+            removeMetadataAndFlatten(jsonObj[key]);
+          }
+        } else {
+          continue;
+        }
+      }
+      return jsonObj;
+    };
+    const flattenedTokens = removeMetadataAndFlatten(tokens);
+
+    return `export default ${JSON.stringify(flattenedTokens, null, 2)
+      .replace(/"([^"]+)":/g, '$1:')
+      .replace('alias', 'aliases')
+      .replace('component', 'components')
+      .replace(/color/g, 'colors')}`;
+  },
```

Comment:
cleanup job for outputing a sanitized string

replacing `alias`, `compoment` & `color` will need to be handled in a safer way